Assets/Scripts/UI/Volunteer/VolunteerWorkProgressBar.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class VolunteerWorkProgressBar : MonoBehaviour
+{
+    [SerializeField]
+    public static VolunteerWorkProgressBar Instance;
+
+    [SerializeField]
+    private GameObject arrow1 = null;
+    [SerializeField]
+    private GameObject arrow2 = null;
+    [SerializeField]
+    private GameObject house = null;
+    [SerializeField]
+    private GameObject bicycle = null;
+    [SerializeField]
+    private GameObject sport = null;
+    [SerializeField]
+    private GameObject sportStockImage = null;
+    [SerializeField]
+    private GameObject coffee = null;
+    [SerializeField]
+    private GameObject coffeeStockImage = null;
+    [SerializeField]
+    private GameObject nature = null;
+    [SerializeField]
+    private GameObject natureStockImage = null;
+
+
+    public void StartAnimation(string animationName)
```

Comment:
Misschien is het handig om hier comments aan toe te voegen over wat de functie doet.